templates/ipfs/home/ipfs/ipfs_default_config

```diff
@@ -194,7 +194,18 @@
     "Router": ""
   },
   "Peering": {
+    {% if ipfs_peering_nodes is defined and ipfs_peering_nodes|length %}
+    "Peers":[
+      {% for node in ipfs_peering_nodes  %}
+      {
+        "ID": node.id,
+        "Addrs": node.addrs
```

Comment:
Does this not need something like `{{ node.addrs | json }}` ?

Reply:
fixed in b23cc85. The whole thing was wrong. added this code by mistake while i was doing something else, but its fixed now